mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Modeling Intersections for Map Navigation
+---
+
+Capturing the various cases of Intersection modelling in OSM and the best practices to follow. The aim of this document is not go into the technicalities of traffic flow in a junction but rather to give a mapper clear cut instructions on how to model different intersections in OSM. Aiming to keep this document simple and easy to follow.
+
+### 1. Intersection of two bi-directional ways
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][efa2ef0b]][efa2ef0b] | [![][ba05460f]][ba05460f] |
+| (_[fullsize][efa2ef0b]_ \| _[wireframe][dbad8b6b] view_) | (_[fullsize][ba05460f]_ \| _[wireframe][a8c293d9] view_)  |
+
+  [efa2ef0b]: ../../images/intersection-modeling/simple_intersection.png "Simple intersection (fullsize view)"
+  [ba05460f]: ../../images/intersection-modeling/simple_intersection_error.png "Incorrect drawing of the simple intersection (fullsize view)"
+  [dbad8b6b]: ../../images/intersection-modeling/simple_intersection_wireframe.png "Simple intersection (wireframe view)"
+  [a8c293d9]: ../../images/intersection-modeling/simple_intersection_wireframe_error.png "Incorrect drawing of the simple intersection (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=20.00/37.75213/-122.48172) 🗺
+
+Each way is a bidirectional road, the movement along which is possible in two directions simultaneously. The point at which the ways intersect denotes the intersection. Crossing ways without a common point make it impossible to build a route between the points that are at the ends of these ways.
+
+### 1a. Intersection and safety islands
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][76c562c6]][76c562c6] | [![][787f1fd3]][787f1fd3] |
+| (_[fullsize][76c562c6]_ \| _[wireframe][b739583e] view_) | (_[fullsize][787f1fd3]_ \| _[wireframe][e0cba39d] view_) |
+
+  [76c562c6]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands.png "Simple intersection with the safety islands (fullsize view)"
+  [787f1fd3]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands_error.png "Incorrect drawing of the simple intersection with the safety islands (fullsize view)"
+  [b739583e]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands_wireframe.png "Simple intersection with the safety islands (wireframe view)"
+  [e0cba39d]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands_wireframe_error.png "Incorrect drawing of the simple intersection with the safety islands (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=20.00/37.75405/-122.48078) 🗺
+
+In the presence of safety islands, located just before the crossroads, there is no need to divide the road onto dual carriageway. Set the nodes for safety islands and tag them as **`traffic_calming=island`**.
```

Comment:
These are known as [traffic islands](https://en.wikipedia.org/wiki/Traffic_island).

Reply:
ty, fixed to `traffic islands`